system-tests/e2e-suite/src/test/java/com/hotels/styx/metrics/ProtocolMetricsTest.java

```diff
@@ -85,19 +71,18 @@ public void stopStyx() {
 
     @AfterMethod
     public void stopOrigins() {
-        nonSslOrigin.stop();
-        sslOrigin.stop();
+        origin.stop();
     }
 
     @Test
     public void recordsServerSideHttp() {
         styxServer = new StyxServer.Builder()
-                .addRoute("/", nonSslOrigin.port())
+                .addRoute("/", origin.port())
                 .start();
 
         FullHttpResponse response = doGet("/");
 
-        assertThat(response.status(), is(OK));
+        assertThat(response.status().code(), is(OK.code()));
```

Comment:
Why was this necessary? If they don't match, it may indicate that the classes are not the same, e.g. Styx `HttpResponseStatus` vs Netty `HttpResponseStatus`.

Reply:
Ok. Importing now Styx `HttpResponseStatus` instead of Netty.